jdbc/src/main/java/io/kestra/jdbc/runner/JdbcScheduler.java

```diff
@@ -45,20 +44,21 @@ public JdbcScheduler(
     public void run() {
         super.run();
 
-        // reset scheduler trigger at end
         executionQueue.receive(
             Scheduler.class,
             execution -> {
-                if (
-                    execution.getTrigger() != null && (
-                        execution.isDeleted() ||
-                            execution.getState().getCurrent().isTerminated()
-                    )
-                ) {
+                // reset scheduler trigger at end
+                if (execution.getTrigger() != null &&
+                    (execution.isDeleted() || execution.getState().getCurrent().isTerminated())) {
                     triggerRepository
                         .findByExecution(execution)
                         .ifPresent(trigger -> triggerRepository.save(trigger.resetExecution()));
                 }
+                else {
+                    triggerRepository
```

Comment:
Should be updated only if a status change

Reply:
Good idea! Done.